.changeset/lazy-papayas-vanish.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
We're moving stuff around and changing implementation a bit (the line where you left the comment) so I'd say this warrants a changelog entry

Reply:
changeset added